test/signers/pss_signer_test.dart

```diff
@@ -0,0 +1,266 @@
+  var sig9b = hex.decode(
+      '80b6d643255209f0a456763897ac9ed259d459b49c2887e5882ecb4434cfd66dd7e1699375381e51cd7f554f2c271704b399d42b4be2540a0eca61951f55267f7c2878c122842dadb28b01bd5f8c025f7e228418a673c03d6bc0c736d0a29546bd67f786d9d692ccea778d71d98c2063b7a71092187a4d35af108111d83e83eae46c46aa34277e06044589903788f1d5e7cee25fb485e92949118814d6f2c3ee361489016f327fb5bc517eb50470bffa1afa5f4ce9aa0ce5b8ee19bf5501b958');*/
+
+  var pubParams = (RSAPublicKey pubk) => () => ParametersWithRandom(
+        PublicKeyParameter<RSAPublicKey>(pubk),
+        FixedSecureRandom(),
+      );
+  var privParams = (RSAPrivateKey privk) => () => ParametersWithRandom(
+        PrivateKeyParameter<RSAPrivateKey>(privk),
+        FixedSecureRandom(),
+      );
+
+  final signer1 = PSSSigner.withSalt(
+    RSAEngine(),
+    Digest('SHA-1'),
+    Digest('SHA-1'),
+    salt: createUint8ListFromHexString(
+        'dee959c7e06411361420ff80185ed57f3e6776af'),
+  );
+
+  /*signer1.init(false, pubParams(pub1)());
+  expect(
+      signer1.verifySignature(
+        createUint8ListFromHexString(
+            'cdc87da223d786df3b45e0bbbc721326d1ee2af806cc315475cc6f0d9c66e1b62371d45ce2392e1ac92844c310102f156a0d8d52c1f4c40ba3aa65095786cb769757a6563ba958fed0bcc984e8b517a3d5f515b23b8a41e74aa867693f90dfb061a6e86dfaaee64472c00e5f20945729cbebe77f06ce78e08f4098fba41f9d6193c0317e8b60d4b6084acb42d29e3808a3bc372d85e331170fcbf7cc72d0b71c296648b3a4d10f416295d0807aa625cab2744fd9ea8fd223c42537029828bd16be02546f130fd2e33b936d2676e08aed1b73318b750a0167d0'),
+        PSSSignature(
+          createUint8ListFromHexString(
+              '9074308fb598e9701b2294388e52f971faac2b60a5145af185df5287b5ed2887e57ce7fd44dc8634e407c8e0e4360bc226f3ec227f9d9e54638e8d31f5051215df6ebb9c2f9579aa77598a38f914b5b9c1bd83c4e2f9f382a0d0aa3542ffee65984a601bc69eb28deb27dca12c82c2d4c3f66cd500f1ff2b994d8a4e30cbb33c'),
+        ),
+      ),
+      isTrue);*/
+
+  test('pss', () {
+    signer1.init(true, privParams(prv1)());
+    final signature1 = signer1.generateSignature(createUint8ListFromHexString(
+        'cdc87da223d786df3b45e0bbbc721326d1ee2af806cc315475cc6f0d9c66e1b62371d45ce2392e1ac92844c310102f156a0d8d52c1f4c40ba3aa65095786cb769757a6563ba958fed0bcc984e8b517a3d5f515b23b8a41e74aa867693f90dfb061a6e86dfaaee64472c00e5f20945729cbebe77f06ce78e08f4098fba41f9d6193c0317e8b60d4b6084acb42d29e3808a3bc372d85e331170fcbf7cc72d0b71c296648b3a4d10f416295d0807aa625cab2744fd9ea8fd223c42537029828bd16be02546f130fd2e33b936d2676e08aed1b73318b750a0167d0'));
+    expect(
+        signature1,
+        equals(PSSSignature(
+          createUint8ListFromHexString(
+              '9074308fb598e9701b2294388e52f971faac2b60a5145af185df5287b5ed2887e57ce7fd44dc8634e407c8e0e4360bc226f3ec227f9d9e54638e8d'
+              '31f5051215df6ebb9c2f9579aa77598a38f914b5b9c1bd83c4e2f9f382a0d0aa3542ffee65984a601bc69eb28deb27dca12c82c2d4c3f66cd500f1'
+              'ff2b994d8a4e30cbb33c'),
+        )));
+    print(hex.encode(signature1.bytes));
```

Comment:
Will update to use the existing test infrastructure after we figure out how to initalise the signer with salts. But these hacky tests actually pass and work properly 🎉